src/models/AppConfig.ts

```diff
@@ -108,16 +108,22 @@ export interface AppUserConfig {
 
 interface BasePromptOptions {
   enabled: boolean;
+}
+
+export interface DelayedPromptOptions extends BasePromptOptions {
   autoPrompt?: boolean;
+  timeDelay?: number;
+  pageViews?: number;
```

Comment:
the reason I left these optional for now is because Typescript is fussy about the types not matching in some other files when running a test (breaks in several places). Will try to fix later